.contributing/the-process.md

```diff
@@ -98,11 +98,21 @@ Create: 
 
 -   An issue on GitHub for each use case
 
-    1.  Add a section label (eg. Auth) 
+1.  Create a tracking issue
```

Comment:
The previous line is Create: ... An issue on GitHub for each use case
- how do these issues relate to each other?  Are they actually the same thing?  

Separate question: - how does the numbered list related to the rest of the dot points?  Is it separate?  It looks strange to me that there's a big list of dot points with a numbered list in the middle without a heading... oh I think I just figured it out - is the numbered list a sublist of the last dot point?  If so, I think you just need to add more indentation - if you're not aware (I wasn't for a long time) the sublist starting point (number or '-' or whatever) needs to start under the first character of the text in the previous line in order to get indented.  So I think you just need an extra space or two and then the indentation will work as you expect.  Or am I off on the wrong path?

Reply:
right path :) ill indent it. thanks for the pickup

Reply:
👍